OAuth/ResourceOwner/FacebookResourceOwner.php

```diff
@@ -67,10 +67,9 @@ public function revokeToken($token)
             'client_secret' => $this->options['client_secret'],
         );
 
-        $response = $this->httpRequest($this->normalizeUrl($this->options['revoke_token_url'], array('token' => $token)), $parameters, array(), HttpRequestInterface::METHOD_POST);
-        $response = $this->getResponseContent($response);
+        $response = $this->httpRequest($this->normalizeUrl($this->options['revoke_token_url'], array('access_token' => $token)), $parameters, array(), HttpRequestInterface::METHOD_DELETE);
 
-        return 'true' == $response;
+        return 200 == $response->getStatusCode();
```

Comment:
From https://github.com/kriswallsmith/Buzz/blob/master/lib/Buzz/Message/Response.php 

```php
    /**
     * Returns the status code of the current response.
     *
     * @return integer
     */
    public function getStatusCode()
    {
        if (null === $this->statusCode) {
            $this->parseStatusLine();
        }
        return $this->statusCode ?: null;
    }
```

You're right, `===` should be used. Unfortunately I don't have time now to work on it, maybe later